src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1697,7 +1697,7 @@ public async Task<bool> HasConfirmedOwnersExceptAsync(Guid organizationId, IEnum
         bool hasOtherOwner = confirmedOwnersIds.Except(organizationUsersId).Any();
         if (!hasOtherOwner && includeProvider)
```

Comment:
`includeProvider` is true by default, and I can't see anywhere that sets it to `false`. I think we could remove it as a parameter, maybe update the name to be `HasOtherOwnersOrProviders` which is a bit more descriptive imo.

Reply:
`ProviderService.RemoveOrganizationAsync` sets `includeProvider: false`

Reply:
Ah good point